roles/galaxyproject.galaxy/defaults/main.yml

```diff
@@ -206,8 +206,8 @@ galaxy_tmpclean_command: "{{ _galaxy_tmpclean_command[(ansible_os_family | lower
 #galaxy_node_version: "XX.YY.Z"
 
 # Where to clone Galaxy from. Backwards compatibility with galaxy_git_repo
-galaxy_repo: "{{ galaxy_git_repo | default('https://github.com/galaxyproject/galaxy.git') }}"
-
+# galaxy_repo: "{{ galaxy_git_repo | default('https://github.com/galaxyproject/galaxy.git') }}"
+galaxy_repo: "https://github.com/BorisYourich/galaxy.git"
```

Comment:
avoid changing vars in the defaults of roles at any cost, define `galaxy_repo` in https://github.com/CESNET/usegalaxy/blob/main/host_vars/galaxy-qa2.galaxy.cloud.e-infra.cz/vars.yml